sdk/src/process_instruction.rs

```diff
@@ -188,6 +226,25 @@ impl BpfComputeBudget {
             max_cpi_instruction_size: 1280, // IPv6 Min MTU size
             cpi_bytes_per_unit: 250,        // ~50MB at 200,000 units
             sysvar_base_cost: 100,
+            bignum_new_base_cost: 100,
```

Comment:
I get the sense that these costs are too small.  By chance have you done any perf measurements of the big-num calls on native?